tests/utils_tests/test_dateparse.py

```diff
@@ -116,6 +116,12 @@ def test_negative(self):
             ('-15:30', timedelta(minutes=-15, seconds=30)),
             ('-1:15:30', timedelta(hours=-1, minutes=15, seconds=30)),
             ('-30.1', timedelta(seconds=-30, milliseconds=-100)),
+            ('-00:01:01', timedelta(minutes=1, seconds=1)),
+            ('-01:01', timedelta(seconds=-59)),
+
+            ('00:-01:-01', None),
+            ('-01:-01', None),
```

Comment:
This is the only case that fails if the patch is reverted.

Reply:
I added this case to test a minus sign after a colon. Can you tell me what should I do to this case?

Reply:
The test is correct. My point was only that tests included with the fix should fail before the patch is applied so that it's clear what's fixed. Additional test coverage (those that already pass without the fix) is okay but should be added in a separate commit or PR.

Reply:
Oh, I got what you mean. I'll split that case into other commits.
Thank you!